Gemfile

```diff
@@ -43,6 +43,16 @@ group :development do
   gem "syntax_tree-haml", require: false
   gem "syntax_tree-rbs", require: false
   gem "web-console"
+
+  # Add speed badges [https://github.com/MiniProfiler/rack-mini-profiler]
+  # gem "rack-mini-profiler"
+
+  # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
+  # gem "spring"
+
+  # For automatic rendering of asciidoc files.
+  gem "guard"
```

Comment:
Ah, that looks nice. I think I'd look to use that in the future but for the moment I'm tempted to just go with the above instead of reworking, if you're ok with it.

Reply:
Yep, it's fine 👍

Reply:
Yep, it's fine 👍

Reply:
I've ended up leaving it out of the final rebase ... less is more, and I'm happy to just use `entr` or `fswatch`.